tools/macos/README.md

```diff
@@ -54,6 +54,24 @@ sudo easy_install pip
 sudo -H pip install docker==2.2.1 ansible==2.5.2 jinja2==2.9.6 couchdb==1.1 httplib2==0.9.2 requests==2.10.0' | bash
 ```
 
+The above section of command installs Oracle JDK 8 as the default Java environment. If you would like to install Open JDK 8 instead of Oracle JDK 8, please run the following section.
+
+```
+echo '
+# install homebrew
+/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+# install for finding alternative versions (Open JDK 8)
+brew tap AdoptOpenJDK/openjdk
+# install Open JDK 8
+brew install adoptopenjdk-openjdk8
+# install scala
+brew install scala
+# install pip
+sudo easy_install pip
+# install script prerequisites
+sudo -H pip install docker==2.2.1 ansible==2.5.2 jinja2==2.9.6 couchdb==1.1 httplib2==0.9.2 requests==2.10.0' | bash
+```
+
```

Comment:
User just needs jdk 8 to be installed, this would be to run tests using gradle, and to build the jar for invoker and controller, we should not have duplicate instructions.
I think giving them one set with openjdk is fine in case they don't have java sdk install.

Reply:
I suggested this - leaving the existing docs for mac in place. Because that’s been something we’ve been using for a while and some are ok with the existing instructions.

Reply:
I’m ok also leaving like it is, and maybe an mention that openjdk 8 could be use. 
What I don’t like is the double instructions